### PR TITLE
fix moveToMy issues

### PR DESCRIPTION
**Pull request by samogot**
- fix range parameter
- do not use in move to source, as moveToMy ignores creeps, so it is impossible for several creeps to harvest

### CODE REVIEW

**Comment by democratic-collaboration** [comment]
[democratic collaboration](https://github.com/TooAngel/democratic-collaboration)
Approved reviews will speed up the merge, request changes will slow it down.

Please review the PR to help.

File: src/prototype_creep_resources.js [hunk 1]
```diff
@@ -446,8 +446,8 @@ Creep.prototype.moveToSource = function(source) {
     this.memory.routing = {};
   }
   this.memory.routing.reverse = false;
-  if (this.room.memory.misplacedSpawn || this.room.controller.level < 3) {
-    this.moveToMy(source.pos);
+  if (this.room.memory.misplacedSpawn || this.room.controller.level < 3 || this.memory.role === 'nextroomer') {
```
Comment (samogot) [inline src/prototype_creep_resources.js]: on restoring high level room they are staying in line instead of harvesting from all available spots near source

Comment (TooAngel) [inline src/prototype_creep_resources.js]: hm, k

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 205/383
    Coefficient: 0.535248041775
    Merging in 3 days 21 hours
    Age 0 days 17 hours

**Pull request merged**